components/Feed/DocumentListContainer.js

```diff
@@ -63,79 +62,71 @@ export const makeLoadMore = ({
     }
   })
 
-class LifecycleWrapper extends Component {
-  componentWillUnmount() {
-    this.props.onComponentWillUnmount && this.props.onComponentWillUnmount()
-  }
-  render() {
-    return this.props.children
-  }
-}
+const DocumentListContainer = props => {
+  const {
+    query,
+    variables,
+    getConnection,
+    mergeConnection,
+    mapNodes,
+    placeholder,
+    help,
+    empty,
+    feedProps,
+    refetchOnUnmount,
+    showTotal
+  } = props
```

Comment:
```suggestion
const DocumentListContainer = ({
    query,
    variables,
    getConnection,
    mergeConnection,
    mapNodes,
    placeholder,
    help,
    empty,
    feedProps,
    refetchOnUnmount,
    showTotal
}) => {
```

If the props object isn't used inside the component, I'd move the deconstruction into the function-parameter